### PR TITLE
Bttajk

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Part of #4273 and #4274 

**Overall change:**
Updates configs and translations for six services

**Code changes:**
- Updates configs for Burmese, Thai, Tigrinya, Aramaic, Japan, and Korea
- Sets values correctly for articles launch
- Updates translations

**Outstanding changes which will be addressed in a later PR**
- Update brand name and opengraph image for Japan
- Burmese translation updates

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/tigrinya.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const service = {
     defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png',
     defaultImageAltText: 'BBC News ትግርኛ',
     dir: `ltr`,
-    externalLinkText: ' ደጋዊ',
+    externalLinkText: ', ደጋዊ',
```
Comment (rhenshaw56) [inline src/app/lib/config/services/tigrinya.js:25]: is there supposed to be a comma here ?

File: src/app/lib/config/services/thai.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ export const service = {
     locale: `th-TH`,
     datetimeLocale: 'th',
     service: 'thai',
-    serviceName: 'News ไทย',
+    serviceName: 'Thai',
```
Comment (sareh) [inline src/app/lib/config/services/thai.js:32]: Looking at the codebase, I don't believe this `serviceName` value is used any longer. (We should remove it). Is there a reason why you're updating it here? Where did you want this value to be used?

Comment (jimjohnsonrollings) [reply src/app/lib/config/services/thai.js:32]: In this case, I’m updating them to be consistent between the services, based on what we set up in Arabic when we launched Articles there.

File: src/app/lib/config/services/tigrinya.js (hunk 1 above)
Comment (jimjohnsonrollings) [reply src/app/lib/config/services/tigrinya.js:25]: Yes I think so. We have a comma here on other services to add a pause between the link text and this word. We only don’t where comma isn’t the right punctuation type for the language (eg Thai)

File: src/app/lib/config/services/thai.js (hunk 2 above)
Comment (sareh) [reply src/app/lib/config/services/thai.js:32]: Okay, I see, thanks.

**Review by sareh — APPROVED** [review]

**Review by 12 — APPROVED** [review]

**Pull request merged**